### PR TITLE
Catch AttributeErrors thrown by ipython_genutils during conversion as ValidationErrors

### PR DESCRIPTION
**Pull request by therzka**
This change allows an error to be thrown and more context to be provided at the time of conversion if a notebook is malformed but isn't caught by validation at read time instead of a generic KeyError

### CODE REVIEW

File: nbformat/converter.py [hunk 1]
```diff
@@ -20,6 +21,17 @@ def convert(nb, to_version):
     to_version : int
         Major revision to convert the notebook to.  Can either be an upgrade or
         a downgrade.
+
+    Raises
```
Comment (gwincr11) [inline nbformat/converter.py:25]: ❇️

**Review by gwincr11 — APPROVED** [review]

File: nbformat/converter.py (hunk 1 above)
Comment (therzka) [reply nbformat/converter.py:25]: ?

Comment (gwincr11) [reply nbformat/converter.py:25]: that is a sparkle, as in nice ✨

Comment (therzka) [reply nbformat/converter.py:25]: just making sure!! <3

**Review by MSeal — APPROVED** [review]

**Comment by MSeal** [comment]
Thanks for adding improvements to the repo. I'm going to merge we need to drop 3.5 anyway

**Pull request merged**